src/components/footer.tsx

```diff
@@ -0,0 +1,147 @@
+import Image from "next/image";
+import { Button } from "./ui/button";
+import Link from "next/link";
+
+export default function Footer() {
+  const footerMenuList = [
+    {
+      header: "KALKULAČKY",
+      items: [
+        {
+          link: "/wages#net-salary",
+          text: "SZČO",
+        },
+        {
+          link: "/wages#net-salary",
+          text: "ČISTÁ MZDA",
+        },
+        {
+          link: "/savings#savings-accounts",
+          text: "ÚROKOVÁ",
+        },
+        {
+          link: "/taxes#dph",
+          text: "DPH ",
+        },
+      ],
+    },
+    {
+      header: "SPORENIE",
+      items: [
+        {
+          link: "/savings#savings-accounts",
+          text: "SPORIACE ÚČTY",
+        },
+        {
+          link: "/wages#net-salary",
+          text: "STAVEBNÉ SPORENIE",
+        },
+        {
+          link: "/savings#retirement-savings",
+          text: " SPORENIE NA DÔCHODOK",
+        },
+        {
+          link: "/investments",
+          text: "INVESTÍCIE ",
+        },
+      ],
+    },
+    {
+      header: "DANE",
+      items: [
+        {
+          link: "/taxes#income-tax",
+          text: "DAŇ Z PRÍJMU",
+        },
+        {
+          link: "/taxes#dph",
+          text: "DPH ",
+        },
+      ],
+    },
+  ];
+
+  return (
+    <footer className="bg-white dark:bg-gray-900">
+      <hr className="my-6 border-gray-200 sm:mx-auto dark:border-gray-700 lg:my-3 w-full max-w-screen-xl" />
+      <div className="mx-auto w-full max-w-screen-xl p-4 py-6 lg:py-8">
+        <div className="md:flex md:justify-between">
+          <div className="flex gap-5 px-6 flex-col pb-10 md:pb-0 md:text-start text-center">
+            <div>
+              <p className="text-gray-500 dark:text-gray-400 font-medium">
+                Objav viac jednoduchých a intuitívnych nástrojov na našej
+                stránke
+              </p>
+            </div>
+            <div className="flex md:justify-start justify-center items-center gap-28 md:pt-6">
+              <Link href="/">
+                <Image src="/logo.svg" alt="Love Logo" width={75} height={75} />
+              </Link>
+              <Button> Prihlásenie</Button>
+            </div>
+          </div>
+          <div className="grid grid-cols-2 gap-8 sm:gap-6 sm:grid-cols-3 text-center md:text-start">
+            {footerMenuList.map((footerSection, key) => (
+              <div key={key}>
+                <h2 className="mb-6 text-sm font-semibold text-gray-900 uppercase dark:text-white">
+                  {footerSection.header}
+                </h2>
+                {footerSection.items.map((item, itemKey) => (
+                  <ul
```

Comment:
Toto ti vygeneruje zlý markup. Aktuálne ti to vypluje 
```
<ul>
    <li>XXXX</li>
</ul>
<ul>
    <li>YYY</li>
</ul>
<ul>
    <li>ZZZ</li>
</ul>
```
ty ale chceš
```
<ul>
    <li>XXXX</li>
    <li>YYY</li>
    <li>ZZZ</li>
</ul>
```

Reply:
<ul> som dal von aby to bolo v poriadku